### DISCUSSION TITLE
How to list tags across multi-instance docs in one place?

### DISCUSSION THREAD

**Discussion by eelmafia** [facebook/docusaurus#10779] [Q&A] [answered]
I'm working on a project that uses multiple doc instances, so I have `/docs/`, `/examples/` and `/community/`. I've noticed that if I set identical tags on a post in `/docs` and `/examples` won't be shown together.  Tags created for content in `/docs` will only encompass `/docs` and vice versa.

Is it possible to have the tags be global so that if `tagX` is shared between multiple docs then clicking on said tag would display links to all sections of the site that have that tag?

**Comment by slorber** [comment]
Unfortunately, we don't have any easy way to achieve that, all plugins are "sandboxed" currently.

In the future, React Server Components might give you the flexibility to pick/merge/render multiple plugin data the way you want, but we don't support that yet (track https://github.com/facebook/docusaurus/issues/9089)

**Reply by eelmafia** [reply]
That's what I assumed. What would be the "hard" way to achieve that?

**Reply by slorber** [reply, marked as answer]
The hard way would be to extend existing plugins (https://github.com/facebook/docusaurus/issues/4138), or fork them to enhance them the way you want.

Alternatively, you can create a plugin that gets access to all the plugins `contentLoaded` data (containing all plugins tags) and implement your own tags merge logic and create a custom tags page from the merged data. See also https://github.com/facebook/docusaurus/issues/10574#issuecomment-2405910732
